CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 
 ## June 2021
 
+- Support custom commit email address for GitHub and GitLab (keep emai private). Thanks to [@philschatz](https://github.com/philschatz) for the contribution! ([#4115](https://github.com/gitpod-io/gitpod/pull/4115))
```

Comment:
nit: typo

Reply:
Also: Since we don't get this into June release, we should move this to July.